src/reporting/market.se

```diff
@@ -568,66 +599,14 @@ def needsMigration():
         return(0)
     if (self.automatedReporterAddress and block.timestamp < self.getAutomatedReportDueTimestamp()):
         return(0)
-    if (self.automatedReporterPayoutDistributionHash and block.timestamp < self.getAutomatedReportDisputeDueTimestamp()):
-        return(0)
-    if (self.automatedReporterPayoutDistributionHash and not self.automatedReporterDisputeBondHolder):
+    if (self.automatedReporterDisputeBondToken and block.timestamp < self.getAutomatedReportDisputeDueTimestamp()):
```

Comment:
I don't believe this change is correct.  We should _not_ migrate if there is an automated report (not dispute) and the dispute time window hasn't passed yet, which is what this check was originally doing.  There also needs to be a check like the one that was deleted:
```
if (self.automatedReportReceived and block.timestamp < self.getAutomatedReportDisputeDueTimestamp()):
    return 0
if (self.automatedReportReceived and not self.automatedReporterDisputeBondToken):
    return 0
```
The second will make it so that if there is an automated report and no dispute _after_ the dispute window then the market should be migrated.